build/generateLicenses.js

```diff
@@ -57,6 +57,7 @@ const licenseJson = execFileSync(
   ],
   {
     encoding: "utf-8",
+    maxBuffer: 1024 * 1024 * 10,
```

Comment:
あ、FIXME書いてあげるのもありかもと思いました！
```suggestion
    maxBuffer: 1024 * 1024 * 10, // FIXME: stdoutではなくファイル出力にする
```